package.yaml

```diff
@@ -13,26 +13,37 @@ extra-source-files:
 description:         Please see the README on GitHub at <https://github.com/kinorius123/snap#readme>
 
 dependencies:
-  - base >= 4.7 && < 5
+  - base >= 4.6 && < 4.10
   - bytestring
   - mtl
   - snap
   - snap-core
   - snap-server
   - snaplet-postgresql-simple
+  - heist
+  - map-syntax
+  - scientific
   - postgresql-simple
   - Decimal
   - transformers
   - lens
   - text
   - time
+  - path-pieces
+  - load-env
```

Comment:
:open_mouth:  @kinorius123 I think it is okay to use that library, so **don't change it**. However, I'd like to let you know that we maintain a library to handle [`.env` files](http://hackage.haskell.org/package/dotenv). :smiley: